packages/lib/src/tabs/Tabs.tsx

```diff
@@ -93,7 +93,7 @@ const DxcTabs = ({
       setTotalTabsWidth(refTabList.current.firstElementChild?.offsetWidth);
       setMinHeightTabs(refTabList?.current?.offsetHeight + 1);
     }
-  }, [childrenArray]);
+  }, [refTabList]);
```

Comment:
Refs do not go inside the dependency array.

Reply:
That's why it was initially changed, the problem here is that in case the tab list is updated, the total width needs to be updated.

Reply:
Totally right. I'm leaving it as a Draft until I fix this.